examples/config files - custom attributes and mappings/1 user-sync-config.yml

```diff
@@ -3,7 +3,8 @@ dashboard:
   # By default, it would look for dashboard-owning-config.yml and
   # dashboard-accessor-*-config.yml in the configuration path,
   # with the yml's identifying the owning organization and accessor organizations
-  # respectively.
+  # respectively. All file references are relative to the folder of the
```

Comment:
You missed this one.  Rephrase as "All relative file paths in any configuration file are taken as relative to that configuration file."